package-lock.json

```diff
@@ -1,8 +1,2614 @@
 {
   "name": "streaming-iterables",
   "version": "6.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I'm not entirely sure why this changed so drastically; I installed `sinon` (to allow me to mock timers) using npm 8.1.0 🤷‍♂️